pkg/destroy/openstack/openstack.go

```diff
@@ -386,7 +386,12 @@ func deleteRouters(opts *clientconfig.ClientOpts, filter Filter, logger logrus.F
 		for _, fip := range allFIPs {
 			_, err := floatingips.Update(conn, fip.ID, floatingips.UpdateOpts{}).Extract()
 			if err != nil {
-				logger.Fatalf("%v", err)
+				// Ignore the error if the resource cannot be found and exit with an appropiate message if it's another type of error
```

Comment:
Nit: "appropiate" -> "appropriate".  Although I think the log messages below make this logic pretty clear, so I'm also fine dropping this comment line entirely.

Reply:
Fixed here and in the other occurrences, thanks.